recipients_app/lib/view/widgets/welcome/phone_input_page.dart

```diff
@@ -45,6 +48,18 @@ class _PhoneInputPageState extends State<PhoneInputPage> {
         child: Column(
           mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
+            Align(
```

Comment:
Instead of 
```
final localizations = AppLocalizations.of(context)!;
localizations.demoCta
```

You can also do:
```
context.l10n.demoCta
```

Reply:
@novas1r1 
If I try to use "context.l10n", then "I10n" is undefined. 🤷🏻‍♀️